packages/backend/src/ee/services/EmbedService/EmbedService.ts

```diff
@@ -232,23 +234,40 @@ export class EmbedService extends BaseService {
         ) {
             throw new ForbiddenError();
         }
+
+        // Require at least one of dashboardUuids or chartUuids
+        if (
+            (!data.dashboardUuids || data.dashboardUuids.length === 0) &&
```

Comment:
what happens if they allow all charts and/or all dashboards ? in that case they don't need to specify any uuids